src/components/new-request/search.vue

```diff
@@ -97,23 +98,22 @@
       </v-col>
 
       <!-- Business Lookup/Fetch -->
-      <v-col v-if="showBusinessLookup" cols="12" md="6">
+      <v-col v-if="showBusinessLookup" cols="12" md="6" class="business-lookup">
         <template v-if="!business">
-          <BusinessLookup v-if="getIsAuthenticated" @business="business=$event"/>
-          <BusinessFetch v-else @business="business=$event"/>
+          <BusinessLookup v-if="getIsAuthenticated" @business="onBusiness($event)"/>
+          <BusinessFetch v-else @business="onBusiness($event)"/>
         </template>
-        <div v-else class="d-flex justify-space-between align-center">
-          <v-text-field
-            append-outer-icon="mdi-close"
-            disabled
-            filled
-            hide-details
-            :value="business.name"
-          />
-          <div @click="business=null">
```

Comment:
Don't need separate clickable icon -- now use readonly text field (instead of disabled) with close icon.